coreos-base/oem-gce/files/units/oem-gce-enable-oslogin.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+DefaultDependencies=false
+Requires=sysinit.target
+After=sysinit.target
+
+[Service]
+ExecStart=/usr/share/oem/scripts/enable-oslogin.sh
```

Comment:
This appears to be running every boot, and the script looks like it will continually be appending to `nsswitch.conf` and overwriting files in `/etc` which the user might have modified.

Reply:
whoops, this was meant to have a `ConditionFirstBoot`. will fix.